docs/source/Evaluation_Tutorial.md

```diff
@@ -251,7 +185,37 @@ docker run \
+
+To train the model, import [**this pre-compiled pipeline**](https://raw.githubusercontent.com/Unity-Technologies/datasetinsights/master/kubeflow/compiled/train_on_synthetic_and_real_dataset.yaml) into your kubeflow cluster. The figure below shows how to do this using the [web UI](https://www.kubeflow.org/docs/pipelines/pipelines-quickstart/#deploy-kubeflow-and-open-the-pipelines-ui). You can optionally use the [KFP CLI Tool](https://www.kubeflow.org/docs/pipelines/sdk/sdk-overview/#kfp-cli-tool).
+
+<img src="_images/kubeflow/upload_pipeline.png" alt="drawing" width="600"/>
+
+Once your pipeline has been imported, you can run it via the web UI as shown below. Alternatively, you can use the [KFP CLI Tool](https://www.kubeflow.org/docs/pipelines/sdk/sdk-overview/#kfp-cli-tool)
+
+<img src="_images/kubeflow/train_on_synthetic_and_real_world_dataset.png" alt="drawing" width="600"/>
+
+You have to specify run parameters required by this pipeline:
+
+- `docker`: Path to a Docker Registry. We suggest changing this parameter to pull our images on Docker Hub with a specific tag, such as `unitytechnologies/datasetinsights:0.2.0`
+- `source_uri`: The dataset source uri. You can use the default value which points to the required dataset for this pipeline.
+- `config`: Estimator config YAML file. You can use the default value which points to a YAML file packaged with our docker images.
+- `checkpoint_file`: Path to the Estimator checkpoint file from previous training runs that you want to load and resume training.
+- `tb_log_dir`: Path to store tensorboard logs used to visualize the training progress.
+- `checkpoint_dir`: Path to store output Estimator checkpoints. You can use one of the checkpoints for estimator evaluation.
+- `volume_size`: Size of the Kubernetes Persistent Volume Claims (PVC) that will be used to store the dataset. You can use the default value.
+
+Set `tb_log_dir` and `checkpoint_dir` to a location that is convenient for you and your Kubernetes cluster has permissions to write to. This is typically a GCS path under the same GCP project. You want to keep a note on these directories that will be used for tensorboard visualization and model evaluation. Note that an invalid location will cause the job to fail, whereas a path to the local filesystem may run but will be hard to monitor as you won't have easy access to the files.
+
+Set `checkpoint_file` to point to an estimator trained in a previous training run on synthetic data. This pipeline will load this model and resume training using real world dataset. You may also use [one of our pre-trained models](#using-our-pre-trained-models) as the starting checkpoint file.
+
+![pipeline graph](_images/kubeflow/train_pipeline_graph.jpg)
+
+Next you can jump back to [part 3](#part-3-monitor-training-in-tensorboard) to monitor the training in tensorboard and continue on to model evaluation.
+
+## Using our pre-trained models
```

Comment:
This section should be moved to a subsection under `## Part 4: Evaluate a model`

Reply:
Done